activerecord/lib/active_record/persistence.rb

```diff
@@ -613,6 +627,15 @@ def update_columns(attributes)
         verify_readonly_attribute(name) || name
       end
 
+      touch = attributes.delete("touch")
+      if touch
+        names = touch if touch != true
+        names = Array.wrap(names)
+        options = names.extract_options!
```

Comment:
Why `extract_options!` here? I don't see any option being passed in the tests.

Reply:
But there is no test coverage for that, so if we remove this line no test will fail. We should add.

Reply:
I will add for this and other methods. Let me know if you want to do by yourself.

Reply:
Please do it. I'm still catching up